### DISCUSSION TITLE
Auth - Quick Search

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#3272] [Enhancements]
while clicking Search Icon the keyboard doesn't popup automatically. Have to click on Search box again for keyboard to come up and start typing. Not a big change, but such minor changes will increase UX.

**Comment by Gobias64** [comment]
I've noticed this issue on both desktop and mobile. The extra click/tap is only a minor annoyance, but I when click on a search icon, I expect that the search box will automatically have focus.

**Comment by Dustysa4** [comment]
This exactly. With latest update on Windows desktop, the cursor is automatically placed in the search field. However on my Android and my iOS, the keyboard does not automatically launch when tapping the search icon. This makes the user tap again, which is really an unnecessary and unintuitive step.

**Comment by Gobias64** [comment]
In the latest version (4.0.2) on macOS, it's still not automatically placing the cursor in the search field.

**Comment by mikaeelg** [comment]
Same issue. Clicking on the search icon places the text cursor in the search field, but typing does nothing. Same issue occurs when "Focus search on app start" is activated. The app's search field does not receive any typing input.
 
Clicking "X" to close the search and then clicking the search icon again does not fix the issue.

The only way to get search to work is to click on the search icon to activate the search field, move Ente app focus to the background by focussing on another program or minimizing Ente to taskbar, and then re-focus Ente to foreground.

OS Version: Windows 10 Home 64-bit 22H2
App Version: 4.0.2

**Comment by ua741** [comment]
This should be fixed in the latest version v4.1.4

**Reply by fizzyfrys** [reply]
I'm still having the issue.
I launch ente auth on windows, type in my windows hello pin, then click the search box (and/or anything else inside ente auth) and it doesn't let me type unless I click off to focus on something else then go back to ente auth. 
I'm on v4.1.6 - Windows 11 24H2
